distribution/docs/src/main/resources/_contents/_app-whitelists/admin-whitelist-contents.adoc

```diff
@@ -0,0 +1,11 @@
+
+=== ${ddf-admin} Whitelist
+
+The following packages have been exported by the ${ddf-admin} application and are approved for use by third parties:
+
+* `org.codice.${ddf-branding-lowercase}.ui.admin.api`
```

Comment:
Should just be `org.codice.ddf...` in all of these. Using the replaceable parameter for something that shouldn't be replaced is dangerous.